src/analytics/analytics-tracker-window.ts

```diff
@@ -1,9 +1,63 @@
+import { isAnalyticsTrackerWindow } from './is-analytics-step-tracker-window';
+
 export interface AnalyticsTracker {
     track(step: string, data: any): void;
-    hit(hitName: string, data: any): void;
-    hasPayloadLimit(data: any): boolean;
 }
 
 export default interface AnalyticsTrackerWindow extends Window {
     analytics: AnalyticsTracker;
 }
+
+interface AnalyticsTrackerWindowGA extends AnalyticsTrackerWindow {
+    ga(command: string, eventName: string, payload: any): void;
```

Comment:
Can you provide a rough shape of the payload, like `Record<string, string>`?

Reply:
done